ManatiCR/FDI
# FDI-31 Endpoint for contact form

[FDI-31](https://manati.atlassian.net/browse/FDI-31)
-
**Steps to test**
- Run `sudo rm -r web/ && composer update && ahoy drush cim -y && ahoy drush cr` smtp module was added
- Run `ahoy docker mailhog-url` and go to the link it displays as response. (There the emails will be sent it, mailhog)
- Go to http://fdi.docker/es/admin/config/emailconfig There edit as you want, one email direction per line. Try to add invalid directions to test it.
- Go to http://fdi.docker/es/graphql/explorer and add the following query to create a contacto node:

Query: 
```
mutation ($input: ContactoInput!) {
  createContacto(input: $input) {
    entity {
      entityId
    }
    errors
    violations {
      message
    }
  }
}
```

Query variable:

```
{
  "input": {
       "title": "Luke Skywalker",
        "field_correo": "luke@gmail.com",
        "field_telefono": "44444444",
        "body": "May the force be with you",
  	"field_codigo_de_seguimient": "frfrf834238d"
    }
}
```
![screenshot from 2018-02-28 12-44-38](https://user-images.githubusercontent.com/9342250/36806287-30a06610-1c85-11e8-87f0-cf8dd8bc276a.png)
- In the mailhog ensure the mail is sent
- Log in as an admministrador de contenido user
- Go to http://fdi.docker/es/admin/fdi/contactos
- There you should be able to go to the node display and only delete the nodes
